libraries/cmake/source/aws-sdk-cpp/README.md

```diff
@@ -1,6 +1,8 @@
-# Linux
+# aws-sdk-cpp library build notes
 
-## Common
+## Linux
+
+### Common
```

Comment:
What's common? And why is it under linux?

Reply:
Normally the subheading under an OS refers to the arch, so common on both the arch we support

Reply:
In this case, was my change incorrect?

Reply:
Sounds like it's fine -- it meets the structure stefano describes.